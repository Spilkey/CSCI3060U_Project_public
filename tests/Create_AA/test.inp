login
user01
create
user05
AA
200logout
exit