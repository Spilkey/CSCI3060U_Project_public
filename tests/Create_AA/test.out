01 user05          AA 000200.00
00